Scheduler: add hint for Scheduled.ApplicationNotRunning skip predicate (#38038)
quarkusio/quarkus: docs/src/main/asciidoc/scheduler-reference.adoc

TIP: A CDI event of type `io.quarkus.scheduler.SkippedExecution` is fired when an execution of a scheduled method is skipped.

> TIP: To skip scheduled executions while Quarkus is shutting down, your can make use of the `Scheduled.ApplicationNotRunning.class` skip predicate.

Reviewer: ```suggestion
TIP: To skip the scheduled executions while the application is starting up/shutting down, you can make use of the `io.quarkus.scheduler.Scheduled.ApplicationNotRunning` skip predicate.
```
